src/main/kotlin/no/nav/emottak/processor/MailProcessor.kt

```diff
@@ -61,10 +60,9 @@ class MailProcessor(
     private suspend fun publishPayloadMessage(payloadMessage: PayloadMessage) {
         with(payloadRepository) {
             fold(
-                block = { insert(payloadMessage.payloads) },
-                recover = { log.error("Could not insert payloads: ${payloadMessage.payloads.map { it }}") },
-                transform = { mailPublisher.publishPayloadMessage(payloadMessage) }
-            )
+                { insert(payloadMessage.payloads) },
+                { log.error("Could not insert payloads: ${payloadMessage.payloads.map { it }}") }
+            ) { mailPublisher.publishPayloadMessage(payloadMessage) }
```

Comment:
Hva skjedde med block, recover og transform? Dette pga. Ktor3 at vi endrer denne? Synes den forrige var mye enklere å lese og forstå, her ser man ikke hva man setter før man går inn på disse ❗ 🤔

Reply:
Dette er helt urelatert til Ktor. Det var en endring jeg gjorde for å _fremheve_ at det er _publish_ - funksjonen som er kjernen i kall-kjeden (løftet lambdaen ut) og repository-funksjonen er en forutsetning.

Reply:
Hvis man ønsker å forbedre leslighet kan man sikkert ta ut funksjonene så de ikke er inline i en annen funksjon; mao så man ikke har funksjoner inni funksjoner inni funksjoner. Synes det fort blir uoversiktlig i Kotlin når man chainer for mange funksjoner og slikt, så sånn sett er det bra å ta ut lambda :+1: